aggsender/aggsender.go

```diff
@@ -197,16 +184,21 @@ func (a *AggSender) Start(ctx context.Context) {
 	if err := a.flow.CheckInitialStatus(ctx); err != nil {
 		a.log.Panicf("error checking flow Initial Status: %v", err)
 	}
-	if a.validator != nil {
-		status, err := a.validator.HealthCheck(ctx)
-		if err != nil {
-			a.log.Warnf("error checking validator health: %v", err)
-		}
-		if !status.IsHealthy() {
-			a.log.Warnf("validator status is not healthy: %s", status.String())
-		}
-		a.log.Infof("validator health check: %s", status.String())
+
+	if !a.cfg.RequireValidatorCall {
+		a.localValidator = validator.NewLocalValidator(
```

Comment:
Add a comment that this is only used for testing and that it should be removed after we test the multisig.

Reply:
Fixed https://github.com/agglayer/aggkit/pull/877/commits/bbff0fe0370caf6c6ad6dc24c282fd56561f1d24